README.md

```diff
@@ -2,6 +2,43 @@
 
 # Smart Social Distancing
 
+- [Smart Social Distancing](#smart-social-distancing)
```

Comment:
What about limiting this table to h1-h4 (not h5). The plugin allows doing that.

I think the table of contents should be relevant and not too long. For example, under Prerequisites, I don't think we need the subsections in this table.

The "Download a sample video" could go under Prerequisites or Installation I think (as optional)

Reply:
Done, I changed the level from h1-h6 to h1-h3.

Reply:
Yeah, that looks much better I think